README.md

```diff
@@ -118,21 +118,21 @@ Once you have the `.env` file configured, run the test using JUnit 5 runner.
 
 ### AzureIdentityCredentialAdapter.ts
 
-The `AzureIdentityCredentialAdapter` class provides a simple bridge to use `DefaultAzureCredential` from `@azure/` package name in `azure-` SDKs. 
+The `AzureIdentityCredentialAdapter` class provides a simple bridge to use `AzureCredential` from `@azure/` package name in `azure-` SDKs. 
```

Comment:
```suggestion
The `AzureIdentityCredentialAdapter` class provides a simple bridge to use a `TokenCredential` based credential from `@azure/` package name in `azure-` SDKs. 
```

Reply:
@jongio looks like the version your are reviewing is not the latest version. Can you review the latest version?

Reply:
Yeah, that one comment isn't valid. The rest are.